app/src/composables/use-revisions.ts

```diff
@@ -88,44 +88,46 @@ export function useRevisions(collection: Ref<string>, primaryKey: Ref<number | s
 				},
 			});
 
-			const createdResponse = await api.get(`/revisions`, {
-				params: {
-					filter: {
-						collection: {
-							_eq: unref(collection),
-						},
-						item: {
-							_eq: unref(primaryKey),
-						},
-						activity: {
-							action: {
-								_eq: 'create',
+			if (!created.value) {
+				const createdResponse = await api.get(`/revisions`, {
+					params: {
+						filter: {
+							collection: {
+								_eq: unref(collection),
+							},
+							item: {
+								_eq: unref(primaryKey),
+							},
+							activity: {
+								action: {
+									_eq: 'create',
```

Comment:
```suggestion
									_eq: Action.CREATE,
```

just for a tad consistency sake